src/core/connection.c

```diff
@@ -4667,6 +4667,7 @@ QuicConnRecvFrames(
         }
 
         case QUIC_FRAME_RESET_STREAM:
+        case QUIC_FRAME_RELIABLE_RESET_STREAM:
```

Comment:
nit: Generally, try to keep `case` statements in order as much as possible. So please put this one after `QUIC_FRAME_STREAM_DATA_BLOCKED`.

Reply:
Sounds good, I'll remember to override my intuition of putting reliable_reset and reset together haha